README.rst

```diff
@@ -0,0 +1,793 @@
+Context
+=======
+
+We build and release software by massively consuming and producing software
+packages such as NPMs, RPMs, Rubygems, etc.
+
+Each package manager, platform, type or ecosystem has its own conventions and
+protocols to identify, locate and provision software packages.
+
+
+Problem
+=======
+
+When tools, APIs and databases process or store multiple package types, it is
+difficult to reference the same software package across tools in a uniform way.
+
+For example, these tools, specifications and API use relatively similar
+approaches to identify and locate software packages, each with subtle
+differences in syntax, naming and conventions:
+
+- Grafeas uses a scheme, namespace, name and version in a URL-like string
```

Comment:
Probably helpful to add links to anything relevant here so folks can easily look at references?

Reply:
Good point. I have added these!